app/models/maintenance_tasks/run.rb

```diff
@@ -32,7 +32,6 @@ class Run < ApplicationRecord
       :cancelled,
     ]
     COMPLETED_STATUSES = [:succeeded, :errored, :cancelled]
-    STUCK_TASK_TIMEOUT = 5.minutes
```

Comment:
I was wondering about using [`#deprecate_constant`](https://www.rubydoc.info/stdlib/core/Module:deprecate_constant), but `MaintenanceTasks::Run` is `@api private`, so I don't think we need to deprecate 👍 (Apps shouldn't be referencing this).